CircuitPython_RPN_Calculator/code.py

```diff
@@ -6,16 +6,13 @@
 from adafruit_display_text.label import Label
 from adafruit_hid.keyboard import Keyboard
 from adafruit_hid.keyboard_layout_us import KeyboardLayoutUS
-from adafruit_hid.keycode import Keycode
 from jepler_udecimal import Decimal, getcontext, localcontext
-import jepler_udecimal.utrig
```

Comment:
It's gross, but this is not actually an unused module.  Importing it causes a "side effect" of registering the trigonometric functions.